recipes/mdaencore/LICENSE

```diff
@@ -0,0 +1,339 @@
+                    GNU GENERAL PUBLIC LICENSE
```

Comment:
Seems like this is part of the source and thus doesn't need to be included here explicitly.

Reply:
Thanks for the review. We had explicitly included the license file because the checklist linked to the example https://github.com/conda-forge/staged-recipes/blob/5eddbd7fc9d1502169089da06c3688d9759be978/recipes/example/meta.yaml#L64-L73 and this seemed to indicate that explicitly including a file was required. But we're happy to follow your guidance and remove it, as it is indeed included in the sources.

Reply:
@IAlibay can you please delete LICENSE? I can't touch the PR. Thanks.

Reply:
Done